### ISSUE TITLE
cuckoo analysis stuck pending web 

### ISSUE THREAD

**Issue by ahmad67** [cuckoosandbox/cuckoo#2045]
Hello dears , when i use cuckoo to analyze files using web interface it keep stuck in pending and does not show any results , also virtualbox does not start automatically as supposed ,also agent.py in guest machine does not show anything it keep like that
![screenshot from 2018-01-05 23-08-21](https://user-images.githubusercontent.com/5900718/34628552-64b040c0-f26d-11e7-8742-3cf7bff469c8.png)

here is my log , and cuckoo.conf , virtualbox.conf,...etc 
log data 
======================================
medusa@Medusais:~$ cuckoo web runserver
Performing system checks...

System check identified no issues (0 silenced).
January 05, 2018 - 22:53:02
Django version 1.8.4, using settings 'cuckoo.web.web.settings'
Starting development server at http://127.0.0.1:8000/
Quit the server with CONTROL-C.
[05/Jan/2018 22:54:56] "GET / HTTP/1.1" 200 27529
[05/Jan/2018 22:54:56] "GET /static/css/vendor.css HTTP/1.1" 304 0
[05/Jan/2018 22:54:56] "GET /static/js/cuckoo/loader.js HTTP/1.1" 304 0
[05/Jan/2018 22:54:56] "GET /static/js/cuckoo/sticky.js HTTP/1.1" 304 0
[05/Jan/2018 22:54:56] "GET /static/js/handlebars-templates.js HTTP/1.1" 304 0
[05/Jan/2018 22:54:56] "GET /static/js/hexdump.js HTTP/1.1" 304 0
[05/Jan/2018 22:54:56] "GET /static/js/vendor.js HTTP/1.1" 304 0
[05/Jan/2018 22:54:56] "GET /static/css/main.css HTTP/1.1" 304 0
[05/Jan/2018 22:54:56] "GET /static/js/cuckoo/process_tree.js HTTP/1.1" 304 0
[05/Jan/2018 22:54:56] "GET /static/js/cuckoo/submission.js HTTP/1.1" 304 0
[05/Jan/2018 22:54:56] "GET /static/js/cuckoo/recent.js HTTP/1.1" 304 0
[05/Jan/2018 22:54:56] "GET /static/js/cuckoo/analysis_network.js HTTP/1.1" 304 0
[05/Jan/2018 22:54:56] "GET /static/js/cuckoo/app.js HTTP/1.1" 304 0
[05/Jan/2018 22:54:56] "GET /static/js/cuckoo/analysis_sidebar.js HTTP/1.1" 304 0
[05/Jan/2018 22:54:56] "GET /static/js/cuckoo/analysis_export.js HTTP/1.1" 304 0
[05/Jan/2018 22:54:56] "GET /static/favicon-32x32.png HTTP/1.1" 304 0
[05/Jan/2018 22:54:56] "GET /static/js/cuckoo/analysis_feedback.js HTTP/1.1" 304 0
[05/Jan/2018 22:54:57] "GET /static/images/close.png HTTP/1.1" 304 0
[05/Jan/2018 22:54:57] "GET /static/images/next.png HTTP/1.1" 304 0
[05/Jan/2018 22:54:57] "GET /static/images/loading.gif HTTP/1.1" 304 0
[05/Jan/2018 22:54:57] "GET /static/graphic/cuckoo_inverse.png HTTP/1.1" 304 0
[05/Jan/2018 22:54:57] "GET /static/images/prev.png HTTP/1.1" 304 0
[05/Jan/2018 22:54:57] "GET /static/graphic/cuckoo-coffee-cup.png HTTP/1.1" 304 0
[05/Jan/2018 22:54:57] "POST /analysis/api/tasks/recent/ HTTP/1.1" 200 13
 Checking for updates...
 You're good to go!

 Our latest blogposts:
 * Cuckoo Sandbox 2.0.5: Office DDE, December 03, 2017.
   Brand new release based on a DDE case study.
   More at https://cuckoosandbox.org/blog/205-office-dde

 * Cuckoo Sandbox 2.0.4, September 06, 2017.
   Introducing Malware Configuration Extraction.
   More at https://cuckoosandbox.org/blog/cuckoo-sandbox-204

 * Cuckoo Sandbox 2.0.0, March 30, 2017.
   First Cuckoo package release!
   More at https://cuckoosandbox.org/blog/cuckoo-sandbox-v2

[05/Jan/2018 22:54:58] "GET /cuckoo/api/status HTTP/1.1" 200 1209
[05/Jan/2018 22:55:12] "POST /submit/api/presubmit/ HTTP/1.1" 302 0
[05/Jan/2018 22:55:12] "GET /submit/pre/6/ HTTP/1.1" 200 20151
[05/Jan/2018 22:55:14] "GET /submit/pre/6/ HTTP/1.1" 200 20151
[05/Jan/2018 22:55:14] "GET /static/css/vendor.css HTTP/1.1" 304 0
[05/Jan/2018 22:55:14] "GET /static/js/hexdump.js HTTP/1.1" 304 0
[05/Jan/2018 22:55:14] "GET /static/js/cuckoo/sticky.js HTTP/1.1" 304 0
[05/Jan/2018 22:55:14] "GET /static/css/main.css HTTP/1.1" 304 0
[05/Jan/2018 22:55:14] "GET /static/js/cuckoo/loader.js HTTP/1.1" 304 0
[05/Jan/2018 22:55:14] "GET /static/js/handlebars-templates.js HTTP/1.1" 304 0
[05/Jan/2018 22:55:14] "GET /static/js/vendor.js HTTP/1.1" 304 0
[05/Jan/2018 22:55:14] "GET /static/js/cuckoo/submission.js HTTP/1.1" 304 0
[05/Jan/2018 22:55:14] "GET /static/js/cuckoo/process_tree.js HTTP/1.1" 304 0
[05/Jan/2018 22:55:14] "GET /static/js/cuckoo/analysis_sidebar.js HTTP/1.1" 304 0
[05/Jan/2018 22:55:14] "GET /static/js/cuckoo/recent.js HTTP/1.1" 304 0
[05/Jan/2018 22:55:14] "GET /static/js/cuckoo/analysis_network.js HTTP/1.1" 304 0
[05/Jan/2018 22:55:14] "GET /static/js/cuckoo/app.js HTTP/1.1" 304 0
[05/Jan/2018 22:55:14] "GET /static/js/cuckoo/analysis_export.js HTTP/1.1" 304 0
[05/Jan/2018 22:55:14] "GET /static/js/cuckoo/analysis_feedback.js HTTP/1.1" 304 0
[05/Jan/2018 22:55:14] "GET /static/favicon-32x32.png HTTP/1.1" 304 0
[05/Jan/2018 22:55:14] "GET /static/images/close.png HTTP/1.1" 304 0
[05/Jan/2018 22:55:14] "GET /static/images/loading.gif HTTP/1.1" 304 0
[05/Jan/2018 22:55:14] "GET /static/graphic/cuckoo_inverse.png HTTP/1.1" 304 0
[05/Jan/2018 22:55:14] "GET /static/images/next.png HTTP/1.1" 304 0
[05/Jan/2018 22:55:14] "GET /static/images/prev.png HTTP/1.1" 304 0
[05/Jan/2018 22:55:15] "GET /static/graphic/cuckoo-coffee-cup.png HTTP/1.1" 304 0
[05/Jan/2018 22:55:15] "POST /submit/api/filetree/ HTTP/1.1" 200 831
[05/Jan/2018 22:55:17] "POST /submit/api/submit HTTP/1.1" 200 32
[05/Jan/2018 22:55:17] "GET /submit/post/6 HTTP/1.1" 200 19865
[05/Jan/2018 22:55:17] "GET /static/css/vendor.css HTTP/1.1" 304 0
[05/Jan/2018 22:55:17] "GET /static/js/cuckoo/sticky.js HTTP/1.1" 304 0
[05/Jan/2018 22:55:17] "GET /static/js/handlebars-templates.js HTTP/1.1" 304 0
[05/Jan/2018 22:55:17] "GET /static/js/hexdump.js HTTP/1.1" 304 0
[05/Jan/2018 22:55:17] "GET /static/css/main.css HTTP/1.1" 304 0
[05/Jan/2018 22:55:17] "GET /static/js/vendor.js HTTP/1.1" 304 0
[05/Jan/2018 22:55:17] "GET /static/js/cuckoo/loader.js HTTP/1.1" 304 0
[05/Jan/2018 22:55:17] "GET /static/js/cuckoo/submission.js HTTP/1.1" 304 0
[05/Jan/2018 22:55:17] "GET /static/js/cuckoo/analysis_sidebar.js HTTP/1.1" 304 0
[05/Jan/2018 22:55:17] "GET /static/js/cuckoo/process_tree.js HTTP/1.1" 304 0
[05/Jan/2018 22:55:17] "GET /static/js/cuckoo/recent.js HTTP/1.1" 304 0
[05/Jan/2018 22:55:17] "GET /static/js/cuckoo/analysis_network.js HTTP/1.1" 304 0
[05/Jan/2018 22:55:17] "GET /static/js/cuckoo/app.js HTTP/1.1" 304 0
[05/Jan/2018 22:55:17] "GET /static/js/cuckoo/analysis_export.js HTTP/1.1" 304 0
[05/Jan/2018 22:55:17] "GET /static/favicon-32x32.png HTTP/1.1" 304 0
[05/Jan/2018 22:55:17] "GET /static/js/cuckoo/analysis_feedback.js HTTP/1.1" 304 0
[05/Jan/2018 22:55:17] "GET /static/images/close.png HTTP/1.1" 304 0
[05/Jan/2018 22:55:17] "GET /static/images/next.png HTTP/1.1" 304 0
[05/Jan/2018 22:55:17] "GET /static/images/loading.gif HTTP/1.1" 304 0
[05/Jan/2018 22:55:17] "GET /static/images/prev.png HTTP/1.1" 304 0
[05/Jan/2018 22:55:17] "GET /static/graphic/cuckoo_inverse.png HTTP/1.1" 304 0
[05/Jan/2018 22:55:17] "GET /static/graphic/cuckoo-coffee-cup.png HTTP/1.1" 304 0
[05/Jan/2018 22:55:17] "POST /analysis/api/tasks/info/ HTTP/1.1" 200 541
[05/Jan/2018 22:55:20] "POST /analysis/api/tasks/info/ HTTP/1.1" 200 541
[05/Jan/2018 22:55:22] "POST /analysis/api/tasks/info/ HTTP/1.1" 200 541
[05/Jan/2018 22:55:25] "POST /analysis/api/tasks/info/ HTTP/1.1" 200 541
[05/Jan/2018 22:55:27] "POST /analysis/api/tasks/info/ HTTP/1.1" 200 541
[05/Jan/2018 22:55:30] "POST /analysis/api/tasks/info/ HTTP/1.1" 200 541
[05/Jan/2018 22:55:32] "POST /analysis/api/tasks/info/ HTTP/1.1" 200 541
[05/Jan/2018 22:55:35] "POST /analysis/api/tasks/info/ HTTP/1.1" 200 541
[05/Jan/2018 22:55:37] "POST /analysis/api/tasks/info/ HTTP/1.1" 200 541
[05/Jan/2018 22:55:40] "POST /analysis/api/tasks/info/ HTTP/1.1" 200 541
[05/Jan/2018 22:55:42] "POST /analysis/api/tasks/info/ HTTP/1.1" 200 541
[05/Jan/2018 22:55:45] "POST /analysis/api/tasks/info/ HTTP/1.1" 200 541
[05/Jan/2018 22:55:47] "POST /analysis/api/tasks/info/ HTTP/1.1" 200 541
[05/Jan/2018 22:55:50] "POST /analysis/api/tasks/info/ HTTP/1.1" 200 541
[05/Jan/2018 22:55:52] "POST /analysis/api/tasks/info/ HTTP/1.1" 200 541
[05/Jan/2018 22:55:55] "POST /analysis/api/tasks/info/ HTTP/1.1" 200 541
[05/Jan/2018 22:55:57] "POST /analysis/api/tasks/info/ HTTP/1.1" 200 541
[05/Jan/2018 22:56:00] "POST /analysis/api/tasks/info/ HTTP/1.1" 200 541
[05/Jan/2018 22:56:02] "POST /analysis/api/tasks/info/ HTTP/1.1" 200 541
[05/Jan/2018 22:56:05] "POST /analysis/api/tasks/info/ HTTP/1.1" 200 541
[05/Jan/2018 22:56:07] "POST /analysis/api/tasks/info/ HTTP/1.1" 200 541
[05/Jan/2018 22:56:10] "POST /analysis/api/tasks/info/ HTTP/1.1" 200 541
[05/Jan/2018 22:56:12] "POST /analysis/api/tasks/info/ HTTP/1.1" 200 541
[05/Jan/2018 22:56:15] "POST /analysis/api/tasks/info/ HTTP/1.1" 200 541

cuckoo.conf 
[cuckoo]
# Enable or disable startup version check. When enabled, Cuckoo will connect
# to a remote location to verify whether the running version is the latest
# one available.
version_check = yes

# If turned on, Cuckoo will delete the original file after its analysis
# has been completed.
delete_original = no

# If turned on, Cuckoo will delete the copy of the original file in the
# local binaries repository after the analysis has finished. (On *nix this
# will also invalidate the file called "binary" in each analysis directory,
# as this is a symlink.)
delete_bin_copy = no

# Specify the name of the machinery module to use, this module will
# define the interaction between Cuckoo and your virtualization software
# of choice.
machinery = virtualbox

# Enable creation of memory dump of the analysis machine before shutting
# down. Even if turned off, this functionality can also be enabled at
# submission. Currently available for: VirtualBox and libvirt modules (KVM).
memory_dump = yes

# When the timeout of an analysis is hit, the VM is just killed by default.
# For some long-running setups it might be interesting to terminate the
# monitored processes before killing the VM so that connections are closed.
terminate_processes = no

# Enable automatically re-schedule of "broken" tasks each startup.
# Each task found in status "processing" is re-queued for analysis.
reschedule = no

# Enable processing of results within the main cuckoo process.
# This is the default behavior but can be switched off for setups that
# require high stability and process the results in a separate task.
process_results = yes
# Limit the amount of analysis jobs a Cuckoo process goes through.
# This can be used together with a watchdog to mitigate risk of memory leaks.
max_analysis_count = 0

# Limit the number of concurrently executing analysis machines.
# This may be useful on systems with limited resources.
# Set to 0 to disable any limits.
max_machines_count = 0

# Limit the amount of VMs that are allowed to start in parallel. Generally
# speaking starting the VMs is one of the more CPU intensive parts of the
# actual analysis. This option tries to avoid maxing out the CPU completely.
max_vmstartup_count = 10

# Minimum amount of free space (in MB) available before starting a new task.
# This tries to avoid failing an analysis because the reports can't be written
# due out-of-diskspace errors. Setting this value to 0 disables the check.
# (Note: this feature is currently not supported under Windows.)
freespace = 1024

# Temporary directory containing the files uploaded through Cuckoo interfaces
# (api.py and Django web interface). Defaults to the default temporary
# directory of the operating system (e.g., /tmp on Linux). Overwrite the value
# if you'd like to specify an alternative path.
tmppath =

# Path to the unix socket for running root commands.
rooter = /tmp/cuckoo-rooter

[feedback]
# Cuckoo is capable of sending "developer feedback" to the developers so that
# they can more easily improve the project. This functionality also allows the
# user to quickly request new features, report bugs, and get in touch with
# support in general, etc.
enabled = no
name =
company =
email =
[resultserver]
# The Result Server is used to receive in real time the behavioral logs
# produced by the analyzer.
# Specify the IP address of the host. The analysis machines should be able
# to contact the host through such address, so make sure it's valid.
# NOTE: if you set resultserver IP to 0.0.0.0 you have to set the option
# `resultserver_ip` for all your virtual machines in machinery configuration.
ip = 192.168.1.8

# Specify a port number to bind the result server on.
port = 8000

# Force the port chosen above, don't try another one (we can select another
# port dynamically if we can not bind this one, but that is not an option
# in some setups)
force_port = no

# Maximum size of uploaded files from VM (screenshots, dropped files, log).
# The value is expressed in bytes, by default 128 MB.
upload_max_size = 134217728
[processing]
# Set the maximum size of analyses generated files to process. This is used
# to avoid the processing of big files which may take a lot of processing
# time. The value is expressed in bytes, by default 128 MB.
analysis_size_limit = 134217728

# Enable or disable DNS lookups.
resolve_dns = yes

# Enable PCAP sorting, needed for the connection content view in the web interface.
sort_pcap = yes

[database]
# Specify the database connection string.
# NOTE: If you are using a custom database (different from sqlite), you have to
# use utf-8 encoding when issuing the SQL database creation statement.
# Examples, see documentation for more:
# sqlite:///foo.db
# postgresql://foo:bar@localhost:5432/mydatabase
# mysql://foo:bar@localhost/mydatabase
# If empty, defaults to a SQLite3 database at $CWD/cuckoo.db.
connection =
# Database connection timeout in seconds.
# If empty, default is set to 60 seconds.
timeout = 60

[timeouts]
# Set the default analysis timeout expressed in seconds. This value will be
# used to define after how many seconds the analysis will terminate unless
# otherwise specified at submission.
default = 120

# Set the critical timeout expressed in (relative!) seconds. It will be added
# to the default timeout above and after this timeout is hit
# Cuckoo will consider the analysis failed and it will shutdown the machine
# no matter what. When this happens the analysis results will most likely
# be lost.
critical = 60

# Maximum time to wait for virtual machine status change. For example when
# shutting down a vm. Default is 60 seconds.
vm_state = 60
========virtualbox.conf======
[virtualbox]
# Specify which VirtualBox mode you want to run your machines on.
# Can be "gui" or "headless". Please refer to VirtualBox's official
# documentation to understand the differences.
mode = headless

# Path to the local installation of the VBoxManage utility.
path = /usr/bin/VBoxManage
# If you are running Cuckoo on Mac OS X you have to change the path as follows:
# path = /Applications/VirtualBox.app/Contents/MacOS/VBoxManage

# Default network interface.
interface = vboxnet0

# Specify a comma-separated list of available machines to be used. For each
# specified ID you have to define a dedicated section containing the details
# on the respective machine. (E.g. cuckoo1,cuckoo2,cuckoo3)
machines = Windows7

[Windows7]
# Specify the label name of the current machine as specified in your
# VirtualBox configuration.
label = Windows7

# Specify the operating system platform used by current machine
# [windows/darwin/linux].
platform = windows

# Specify the IP address of the current virtual machine. Make sure that the
# IP address is valid and that the host machine is able to reach it. If not,
# the analysis will fail.
ip = 192.168.56.101

# (Optional) Specify the snapshot name to use. If you do not specify a snapshot
# name, the VirtualBox MachineManager will use the current snapshot.
# Example (Snapshot1 is the snapshot name):
snapshot = snapshot1

# (Optional) Specify the name of the network interface that should be used
# when dumping network traffic from this machine with tcpdump. If specified,
# overrides the default interface specified in auxiliary.conf
# Example (vboxnet0 is the interface name):
interface = vboxnet0

# (Optional) Specify the IP of the Result Server, as your virtual machine sees it.
# The Result Server will always bind to the address and port specified in cuckoo.conf,
# however you could set up your virtual network to use NAT/PAT, so you can specify here
# the IP address for the Result Server as your machine sees it. If you don't specify an
# address here, the machine will use the default value from cuckoo.conf.
# NOTE: if you set this option you have to set result server IP to 0.0.0.0 in cuckoo.conf.
# Example:
resultserver_ip = 0.0.0.0

# (Optional) Specify the port for the Result Server, as your virtual machine sees it.
# The Result Server will always bind to the address and port specified in cuckoo.conf,
# however you could set up your virtual network to use NAT/PAT, so you can specify here
# the port for the Result Server as your machine sees it. If you don't specify a port
# here, the machine will use the default value from cuckoo.conf.
# Example:
resultserver_port =2042
# (Optional) Set your own tags. These are comma separated and help to identify
# specific VMs. You can run samples on VMs with tag you require.
#tags =

# Mostly unused for now. Please don't fill it out.
#options =

# (Optional) Specify the OS profile to be used by volatility for this
# virtual machine. This will override the guest_profile variable in
# memory.conf which solves the problem of having multiple types of VMs
# and properly determining which profile to use.
#osprofile =
[honeyd]
# For more information on this VM please refer to the "services" section of
# the conf/auxiliary.conf configuration file. This machine is a bit special
# in the way that its used as an additional VM for an analysis.
# *NOTE* that if this functionality is used, the VM should be registered in
# the "machines" list in the beginning of this file.
label = honeyd
platform = linux
ip = 192.168.56.102
# The tags should at least contain "service" and the name of this service.
# This way the services auxiliary module knows how to find this particular VM.
tags = service, honeyd
# Not all services actually have a Cuckoo Agent running in the VM, for those
# services one can specify the "noagent" option so Cuckoo will just wait until
# the end of the analysis instead of trying to connect to the non-existing
# Cuckoo Agent. We can't really intercept any inter-VM communication from the
# host / gateway so in order to dump traffic between VMs we have to use a
# different network dumping approach. For this machine we use the "nictrace"
# functionality from VirtualBox (which is basically their internal tcpdump)
# and thus properly dumps inter-VM traffic.
options = nictrace noagent

**Comment by doomedraven** [comment]
i don't see any task upload in logs

**Comment by ahmad67** [comment]
i don't know why it not shown in logs , maybe that why the status still pending? , what is the problem ?  
i run this command cuckoo web runserver , then drag and drop the .exe file , after that press analysis and you know the problem after that

**Comment by jbremer** [comment]
The Agent doesn't emit any output (anymore). Does the analysis itself succeed?

**Comment by jbremer** [comment]
Closing issue for lack of information, feel free to reopen if you have new information.
